src/runners/baseRunner/RunnerResultAnalyzer.ts

```diff
@@ -55,10 +54,6 @@ export abstract class RunnerResultAnalyzer {
                             this.testMessageLocation = new Location(currentItem.uri, new Range(currentItem.range.start.line, 0, currentItem.range.start.line, 0));
                         }
                     }
-                    if (assertionFailure) {
-                        assertionFailure.location = this.testMessageLocation;
-                        setTestState(this.testContext.testRun, currentItem, TestResultState.Failed, assertionFailure);
-                    }
```

Comment:
1: stop adding the assertionFailure during stack-trace processing, since we do not know whether future entries contain a more precise location to create the message at